addons/pos_restaurant/static/src/app/floor_screen/table.js

```diff
@@ -35,13 +35,6 @@ export class Table extends Component {
             const minWidth = 100 + 20;
             const nbrHorizontal = Math.floor(window.innerWidth / minWidth);
             const widthTable = (window.innerWidth - nbrHorizontal * 10) / nbrHorizontal;
-            const position_h =
-                widthTable * (index % nbrHorizontal) + 5 + (index % nbrHorizontal) * 10;
-            const position_v =
-                widthTable * Math.floor(index / nbrHorizontal) +
-                5 +
-                Math.floor(index / nbrHorizontal) * 10;
-            const widthTable = (window.innerWidth - nbrHorizontal * 10) / nbrHorizontal;
```

Comment:
this was added in the previous commit, not it's being removed. It's better if you remove the diff in the previous commit.

Reply:
There is an easy way to edit / amend on the first commit ? Thx

Reply:
I don't know of easy way. I normally do `rebase -i` to amend a commit not on the top.